src/containers/urls.js

```diff
@@ -0,0 +1,9 @@
+const base = 'https://ah-backend-groot.herokuapp.com/api'
+
+const endPoints = {
+getCategories: `${base}/api/categories/?limit=6`,
+articles: `${base}/api/articles/?limit=6`,
```

Comment:
@walimike , when slash 'api' in the base dont include it in the endpoints as well. It will break our code.